src/js/libcs/Operators.js

```diff
@@ -2775,8 +2775,8 @@ eval_helper.shapeconvert = function(a) {
         var mx = aa.value[0].value.real;
         var my = aa.value[1].value.real;
         var r = a.value.value[1].value.real;
-        li = [];
-        var n = 36;
+        var n = 125;
+        li = new Array(n);
```

Comment:
OK, `new Array(n)` seems to beat `[]` at least in Chrome and Firefox, see http://jsperf.com/cindyjs-new-array-n-vs/3